.cmake-format.json

```diff
@@ -0,0 +1,83 @@
+{
+    "parse": {
+	"additional_commands": {
+            "CheckIPProto": {
+		"kwargs": {
+                    "_proto": "*"
+		}
+            },
+            "CheckType": {
+		"kwargs": {
+                    "_type": "*",
+                    "_alt_type": "*",
+                    "_var": "*"
+		}
+            },
+	    "SetPackageVersion": {
+		"kwargs": {
+		    "_version": "*"
+		}
+	    },
+	    "SetPackageFileName": {
+		"kwargs": {
+		    "_version": "*"
+		}
+	    },
+	    "SetPackageInstallScripts": {
+		"kwargs": {
+		    "VERSION": "*"
+		}
+	    },
+	    "ConfigurePackaging": {
+		"kwargs": {
+		    "_version": "*"
+		}
+	    },
+	    "SetPackageGenerators": {},
+	    "SetPackageMetadata": {},
+	    "FindRequiredPackage": {
+		"kwargs": {
+		    "packageName": "*"
+		}
+	    },
+	    "InstallClobberImmune": {
+		"kwargs": {
+		    "_srcfile": "*",
+		    "_dstfile": "*"
+		}
+	    },
+	    "InstallPackageConfigFile": {
+		"kwargs": {
+		    "_srcfile": "*",
+		    "_dstdir": "*",
+		    "_dstfilename": "*"
+		}
+	    },
+	    "InstallShellScript": {
+		"kwargs": {
+		    "_srcfile": "*",
+		    "_dstfile": "*"
+		}
+	    },
+	    "InstallSymLink": {
+		"kwargs": {
+		    "_filepath": "*",
+		    "_sympath": "*"
+		}
+	    }
+	}
+    },
+    "format": {
+	"line_width": 100,
+	"tab_size": 4,
+	"separate_ctrl_name_with_space": true,
+	"max_subgroups_hwrap": 3,
+	"line_ending": "unix"
+    },
+    "markup": {
+	"enable_markup": false
+    },
+    "lint": {
```

Comment:
These are options for cmake-lint which you currently do not need and could just drop.

 If you [added cmake-lint to `.pre-commit.yaml` as well](https://github.com/zeek/spicy/blob/90d4009b290ac00d1f948a5464475a5d26ac53b5/.pre-commit-config.yaml#L73) it will currently flag a lot of style issues and the most user-visible one (apart from missing docstrings [C0111](https://cmake-format.readthedocs.io/en/latest/lint-implemented.html#c0111)) are probably around missing `COMMENT`s in e.g., custom commands ([C0113](https://cmake-format.readthedocs.io/en/latest/lint-implemented.html#c0113)); fixing these usually leads to better log messages during the build.

Reply:
I opted to drop it.